examples/cpp/put-get-remove/main.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <iostream>
+
+// Include the Geode libraries.
+#include <geode/CacheFactory.hpp>
+#include <geode/PoolManager.hpp>
+
+// Use the "geode" namespace.
+using namespace apache::geode::client;
+
+
+int main(int argc, char** argv) {
+
+  auto cacheFactory = CacheFactory();
+  cacheFactory.set("log-level", "none");
+  auto cache = cacheFactory.create();
+
+  auto poolFactory = cache.getPoolManager().createFactory();
+  poolFactory->addLocator("localhost", 10334);
+  auto pool = poolFactory->create("pool");
+
+  auto regionFactory = cache.createRegionFactory(PROXY);
+
+  auto region = regionFactory.setPoolName("pool").create("orders");
+
+
+  std::cout << "Storing orders in the region" << std::endl;
+  region->put("a123","1983");
+  region->put("a124","1982");
```

Comment:
I don't think these value are very useful. It too me a second to even recognize the difference between each line. You are giving a false impression that only strings can be stored and that integers should be represented as strings. 

```
region->put("one", "first");
region->put("two", "second");
```
or even:
```
region->put("key-1", "value-1");
```
or:
```
region->put("string", "a string");
region->put("int", 1234);
```